specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -348,6 +348,14 @@ model Test {
   @added(APIVersions.v2024_07_01_preview)
   metricsReferenceIdentityId?: string;
 
+  @doc("Type of the managed identity built in load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityType?: ManagedIdentityType = ManagedIdentityType.SystemAssigned;
+
+  @doc("Resource Ids of the managed identity built in to load test engines. Required if engineBuiltinIdentityType is UserAssigned.")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityIds?: string[];
```

Comment:
I think you should use BuiltIn with a capital I because Build In is 2 words.

Reply:
I agree with Jeff -- "Builtin" should be "BuiltIn" with a capital I.

Reply:
done